sdk/src/androidTest/java/com/podio/sdk/domain/ItemTest.java

```diff
@@ -27,7 +27,7 @@ public void testAddUnverifiedValue() {
         item.addValue("FIELD-1", new TextField.Value("VALUE"));
 
         Gson gson = new Gson();
-        String actualJson = gson.toJson(item.getCreateData());
+        String actualJson = gson.toJson(item.getCreateData(false));
```

Comment:
I had to update the getCreateData method on the Item object to take an extra parameter called hasLinkedAccountData, if false it uses the default body while making a request or else it adds linked_account parameter to the request's body.